Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java

```diff
@@ -1,30 +1,28 @@
-
-package org.openas2.processor.receiver.api;
-
-import java.util.List;
-
-import org.openas2.message.AS2Message;
-import org.openas2.processor.receiver.MessageBuilderModule;
-
-public class AS2FileReceiverModule extends MessageBuilderModule {
-
-    protected AS2Message createMessage() {        
-        return new AS2Message();
-    }
-
-	public void doStart() {
-		return;
-	}
-
-	public void doStop() {
-		return;
-	}
-
-	@Override
-	public boolean healthcheck(List<String> failures)
-	{
-		// TODO Auto-generated method stub
-		return true;
-	}
-
-}

```

Comment:
I didn't see the purpose of this file. Seems like left over from an unfinished change. I recommend we remove it from the master branch.

Reply:
I will leave it as it is.Within this PR do formatting and remove comments if needed. No changes. Sorry.

Reply:
Agreed. This PR is purely for reformatting.